devnet/bridge.yaml

```diff
@@ -42,7 +42,16 @@ spec:
         - name: guardiand
           image: guardiand-image
           command:
+# Uncomment this for in-place debugging using dlv
```

Comment:
Why is this commented out? You could always run it with delve and use `--continue` to not block and wait for a debugger to be attached (for anyone trying to debug right from the start, this could then be commented out)?